joint_trajectory_controller/test/test_trajectory.cpp

```diff
@@ -98,78 +102,87 @@ TEST(TestTrajectory, sample_trajectory_positions)
   const rclcpp::Time time_now = rclcpp::Clock().now();
   auto traj = joint_trajectory_controller::Trajectory(time_now, point_before_msg, full_msg);
 
-  trajectory_msgs::msg::JointTrajectoryPoint expected_state;
+  trajectory_msgs::msg::JointTrajectoryPoint output_point;
   joint_trajectory_controller::TrajectoryPointConstIter start, end;
+  rclcpp::Duration period = rclcpp::Duration::from_nanoseconds(1e7);
+  bool do_ruckig_smoothing = true;
 
   double duration_first_seg = 1.0;
   double velocity = (p1.positions[0] - point_before_msg.positions[0]) / duration_first_seg;
 
   // sample at trajectory starting time
   {
-    traj.sample(time_now, expected_state, start, end);
+    traj.sample(time_now, start, period, do_ruckig_smoothing, output_point, end);
```

Comment:
Other than renaming/adjusting tests, is there a possibility to actually test for Jerk limits?